### PR TITLE
Policy location mentioned

### PR DESCRIPTION
**Pull request by joinimran**
Policy location mentioned as it was missing. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5737

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/security-policy-settings/interactive-logon-machine-inactivity-limit.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Describes the best practices, location, values, management, and security conside
 
 ## Reference
 
-Beginning with Windows Server 2012 and Windows 8, Windows detects user-input inactivity of a sign-in (logon) session by using the security policy setting **Interactive logon: Machine inactivity limit**. If the amount of inactive time exceeds the inactivity limit set by this policy, then the user’s session locks by invoking the screen saver (screen saver should be active on the destination machine). This policy setting allows you to control the locking time by using Group Policy.
+Beginning with Windows Server 2012 and Windows 8, Windows detects user-input inactivity of a sign-in (logon) session by using the security policy setting **Interactive logon: Machine inactivity limit**. If the amount of inactive time exceeds the inactivity limit set by this policy, then the user’s session locks by invoking the screen saver (screen saver should be active on the destination machine. You can activate screen saver by enableing Group Policy at User Configuration\Administrative Templates\Control Panel\Personalization\ "Enable screen saver"). This policy setting allows you to control the locking time by using Group Policy.
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/interactive-logon-machine-inactivity-limit.md]:
```suggestion
Beginning with Windows Server 2012 and Windows 8, Windows detects user-input inactivity of a sign-in (logon) session by using the security policy setting **Interactive logon: Machine inactivity limit**. If the amount of inactive time exceeds the inactivity limit set by this policy, then the user’s session locks by invoking the screen saver (screen saver should be active on the destination machine. You can activate the screen saver by enabling the Group Policy **User Configuration\Administrative Templates\Control Panel\Personalization\Enable screen saver**). This policy setting allows you to control the locking time by using Group Policy.
```
- enableing -> enabling
- joined the GPO "Enable screen saver" to the path of the object
- marked the full path including object with bold text format
- added definite article "the" in a couple of places where it was missing

**Comment by joinimran** [comment]
@mypil FYI Please, looks like its ready to go live.

**Comment by mypil** [comment]
@JohanFreelancer9 - Kindly confirm and do the copy/editor review for this PR. Thank you.

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #5737.

Thank you.

cc: @kenwith

File: windows/security/threat-protection/security-policy-settings/interactive-logon-machine-inactivity-limit.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Describes the best practices, location, values, management, and security conside
 
 ## Reference
 
-Beginning with Windows Server 2012 and Windows 8, Windows detects user-input inactivity of a sign-in (logon) session by using the security policy setting **Interactive logon: Machine inactivity limit**. If the amount of inactive time exceeds the inactivity limit set by this policy, then the user’s session locks by invoking the screen saver (screen saver should be active on the destination machine). This policy setting allows you to control the locking time by using Group Policy.
+Beginning with Windows Server 2012 and Windows 8, Windows detects user-input inactivity of a sign-in (logon) session by using the security policy setting **Interactive logon: Machine inactivity limit**. If the amount of inactive time exceeds the inactivity limit set by this policy, then the user’s session locks by invoking the screen saver (screen saver should be active on the destination machine. You can activate the screen saver by enabling the Group Policy **User Configuration\Administrative Templates\Control Panel\Personalization\Enable screen saver**). This policy setting allows you to control the locking time by using Group Policy.
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/interactive-logon-machine-inactivity-limit.md]: Sorry, I overlooked where a closing parenthesis belonged in the modified text:
```suggestion
Beginning with Windows Server 2012 and Windows 8, Windows detects user-input inactivity of a sign-in (logon) session by using the security policy setting **Interactive logon: Machine inactivity limit**. If the amount of inactive time exceeds the inactivity limit set by this policy, then the user’s session locks by invoking the screen saver (screen saver should be active on the destination machine). You can activate the screen saver by enabling the Group Policy **User Configuration\Administrative Templates\Control Panel\Personalization\Enable screen saver**. This policy setting allows you to control the locking time by using Group Policy.
```

Comment (illfated) [reply windows/security/threat-protection/security-policy-settings/interactive-logon-machine-inactivity-limit.md]: The closing parenthesis is moved to the end of this sentence: (screen saver should be active on the destination machine).

**Pull request merged**